src/EditorFeatures/CSharpTest/ChangeSignature/ChangeSignature_CheckAllSignatureChanges.cs

```diff
@@ -148,7 +149,7 @@ void M()
     void Goo5(int a, string b, bool c) { }
 }";
             var signaturePartCounts = new[] { 0, 3, 0, 0 };
-            await TestAllSignatureChangesAsync(LanguageNames.CSharp, markup, signaturePartCounts);
+            await TestAllSignatureChangesAsync(LanguageNames.CSharp, markup, signaturePartCounts, new CSharpParseOptions(LanguageVersion.CSharp7));
```

Comment:
>new CSharpParseOptions(LanguageVersion.CSharp7) [](start = 98, length = 47)

Consider adding a prototype comment, we might want this test to target default options in the future.